Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -14,9 +14,9 @@
     '2018Design',
     '2018DesignPU',
     '2019',
-#    '2019PU',
+    '2019PU',
```

Comment:
@boudoul I guess this line with the corresponding changed order is the responsible of the visible difference in the standard matrix:

```
< 12024.0 TTbar_13TeV_TuneCUETP8M1_2019Design_GenSimFull+DigiFull_2019Design+RecoFull_2019Design+HARVESTFull_2019Design 
---
> 12024.0 TTbar_13TeV_TuneCUETP8M1_2019PU_GenSimFull+DigiFullPU_2019PU+RecoFullPU_2019PU+HARVESTFullPU_2019PU+NanoFull_2019PU 
```

Not a problem in itself, just to be aware of it

Reply:
Hi @boudoul @fabiocos yes because this number is related to TTbar Design in IB as you can see at  
https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_2017.py

When you change https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py, moving the comment of line 17, you will change the order of the wfs numbers.

I remember we have a problem like this in the past and to keep all under control @kpedro88  has changed something at 
numWFConflict = [[11800,12000],[12200,12400],[20800,21200],[21600,22400],[25000,26000],[50000,51000]]

Maybe increasing the range [11800,12000] would work.

Reply:
ok yes I will try to keep the WF numbers unchanged  in a next commit later today

Reply:
If you are activating the 2019 PU workflows, you should remove the ranges [11800,12000],[12200,12400] from the conflict list. These were reserved for that purpose.

Reply:
ah you're right- I missed your comment  I just re-oreder to preserve but actually what you wrote makes more sense - Let me fix in a next commit